src/realm/exec/CMakeLists.txt

```diff
@@ -31,12 +31,19 @@ set_target_properties(RealmBrowser PROPERTIES
 )
 target_link_libraries(RealmBrowser Storage)
 
-add_executable(Realm2JSON EXCLUDE_FROM_ALL realm2json.cpp )
-set_target_properties(Realm2JSON PROPERTIES
-    OUTPUT_NAME "realm2json-6"
-    DEBUG_POSTFIX ${CMAKE_DEBUG_POSTFIX}
-)
-target_link_libraries(Realm2JSON Storage)
+if(NOT CMAKE_SYSTEM_NAME MATCHES "^Windows")
```

Comment:
Why do we not want to build this on windows?

Reply:
It looks like it failed here: https://ci.realm.io/blue/organizations/jenkins/realm%2Frealm-core/detail/PR-4165/1/pipeline/